content/docs/1_guide/16_emails/guide.txt

```diff
@@ -277,6 +277,8 @@ $kirby->email([
     // …
 		'template' => 'email',
     'beforeSend' => function ($mailer) {
+				// $image should return file path
+				$image = asset('assets/images/logo.png')->root();
         $mailer->AddEmbeddedImage($image, 'image', 'image.png', 'base64', 'image/png');
```

Comment:
I think that's a good addition, but then it could even make use of more asset methods. 

```suggestion
				// $image should return file path
				$image = asset('assets/images/logo.png')->root();
        $mailer->AddEmbeddedImage($image, 'image', $image->filename(), 'base64', $image->mime());
```

Reply:
Indeed!